frontend/src/scenes/retention/RetentionTable.tsx

```diff
@@ -167,12 +168,25 @@ export function RetentionTable({ dashboardItemId = null }: { dashboardItemId?: n
                                                     <tr key={personAppearances.person.id}>
                                                         <td className="text-overflow" style={{ minWidth: 200 }}>
                                                             <Link
-                                                                to={urls.person(
-                                                                    personAppearances.person.distinct_ids[0]
-                                                                )}
+                                                                to={
+                                                                    isGroupType(personAppearances.person)
+                                                                        ? urls.group(
+                                                                              String(
+                                                                                  personAppearances.person
+                                                                                      .group_type_index
+                                                                              ),
+                                                                              personAppearances.person.group_key
+                                                                          )
+                                                                        : urls.person(
+                                                                              personAppearances.person.distinct_ids[0]
+                                                                          )
+                                                                }
                                                                 data-attr="retention-person-link"
                                                             >
-                                                                {personAppearances.person.name}
+                                                                {/* TODO: use groupDisplayId */}
```

Comment:
Both of these are displayed wrong.

For groups, we should use the method introduced in https://github.com/PostHog/posthog/pull/7600
For persons, we should cosnsitently use `asDisplay`

Not sure what the todo is about as well

Reply:
oh the comment was the function in #7600 just didn't merge master yet to have access to it